samples/interactive-tutorials/product/remove_fulfillment_places.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2021 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START retail_remove_fulfillment_places]
+import datetime
+import os
+import time
+
+from google.api_core.client_options import ClientOptions
+from google.cloud.retail import ProductServiceClient, \
+    RemoveFulfillmentPlacesRequest
+
+from setup.setup_cleanup import create_product, get_product
+
+project_number = os.getenv('GOOGLE_CLOUD_PROJECT_NUMBER')
+endpoint = "retail.googleapis.com"
+product_id = "remove_fulfillment_test_product_id"
```

Comment:
Let's use a random id, which will make the test more stable.

Reply:
in the tutorial we would like to show the feature, that if add/remove fulfillment places is requested with outdated timestamp then the inventory information will not be changed. 
so the flow in the tutorials is the following:
1. Create Product with fulfillment places
2. Remove some of places using current timestamp
Check the places were removed
3. disable create_product() and change the time in the add/remove fulfillment places request
4. enable the delete_product() call
5. run the sample again
6. Check that the places were NOT removed

For that reason I have used the same product_id - to make it possible to run the sample several times.

We could change the tutorial flow and just run the sample only once requesting the add/remove fulfillment places two times. But in this case the output in the Terminal will be pretty long, and I am not sure if it will be better for user

If you suggest to call add/remove_fulfillment_places twice during one run, I will rewrite the tutorial and update the sample.

Reply:
Yes I think the biggest issue with this flow is that it'll break the CI test, if the test is automatically running for a second time. On the second run, the create product API will fail, and "assert re.match(".*product is created.*", output)" will break.

About "if add/remove fulfillment places is requested with outdated timestamp then the inventory information will not be changed", can we just have users to uncomment the second outdated request timestamp and run the sample for a second time? Long output maybe an issue but we may print some breakers to make it more clear.

Reply:
fixed